app/controllers/application_controller.rb

```diff
@@ -26,4 +33,13 @@ def check_authorization
     flash[:warn] = t('shared.errors.authorisation')
     redirect_to root_path
   end
+
+  private
+
+  def sign_out_user
+    UserSignOutEvent.create(user_id: warden.user.user_id)
```

Comment:
could we have a new TimeOut event? so we can see how often people timeout?

Reply:
Timeout event created